instrumentation/trilogy/test/opentelemetry/instrumentation/trilogy/instrumentation_test.rb

```diff
@@ -9,6 +9,16 @@
 require_relative '../../../../lib/opentelemetry/instrumentation/trilogy'
 require_relative '../../../../lib/opentelemetry/instrumentation/trilogy/patches/client'
 
+# This test suite requires a running mysql container and dedicated test container. We can use the same
+# docker-compose file as the mysql2 instrumentation tests. The test container should have the mysql client.
```

Comment:
```suggestion
# docker-compose service as the mysql2 instrumentation tests. The test container should run the mysql client.
```

Reply:
Thank you @kaylareopelle! Great suggestions.